packages/form-js-viewer/src/render/components/form-fields/parts/Datepicker.js

```diff
@@ -51,6 +51,7 @@ export default function Datepicker(props) {
   useEffect(() => {
 
     if (disabled) {
+      flatpickrInstance && flatpickrInstance.setDate(null, true);
```

Comment:
This shows the placeholder in any case when disabled, also when valid input data was given.

![image](https://user-images.githubusercontent.com/9433996/212063902-9cec2d97-365b-4a69-ad5e-f22d9d4acc22.png)

![Bildschirmfoto 2023-01-12 um 13 13 47](https://user-images.githubusercontent.com/9433996/212063850-98b40908-f719-4dec-8127-748ffc36dca0.png)

Reply:
This is inconsistent with other fields, where we display the initial value even on disabled.

![image](https://user-images.githubusercontent.com/9433996/212064932-2d6c3e51-b0c2-4eb9-8a2a-1e2e2eea3089.png)

Reply:
Wait this confuses me a bit. Is the goal of 'disabled' to work like a read-only then?

Reply:
I think a few components need adjustment then

Reply:
It's not `readonly` as it isn't submitted. However, it should be displayed if data is imported (cf. https://bpmn.io/blog/posts/2022-form-js-default-values-improved-submit.html).

Reply:
> I think a few components need adjustment then

Indeed it seems like this isn't working for the `taglist` as well.
![image](https://user-images.githubusercontent.com/9433996/212068913-009e5ad6-33cb-4caa-8c7f-cdb7134650fb.png)
![image](https://user-images.githubusercontent.com/9433996/212068961-0508231a-dd56-4976-8f26-134f9d258492.png)

I'll create an issue for that.

Reply:
Not that big of a deal, as we should discourage the use of `disabled` fields I believe (cf. #362)

Reply:
Here you go: https://github.com/bpmn-io/form-js/issues/496